deploy/vagrant/conf/tachyon.yml

```diff
@@ -1,13 +1,16 @@
-# Type can be {Local|Github|Release}
-# Local means using the Tachyon repo which this directory resides in
-# Release means using binary distribution from https://github.com/amplab/tachyon/releases
-# Github means cloning and compiling github repo
+# {Github | Local | Release}
```

Comment:
the previous comments were useful. What does 'local' mean? Does it simply copy the local files to the machines?